modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -0,0 +1,760 @@
+/* global __filename, RTCIceGatherer, RTCIceTransport */
+
+import { getLogger } from 'jitsi-meet-logger';
+import yaeti from 'yaeti';
+
+import { InvalidStateError } from './errors';
+
+const logger = getLogger(__filename);
+
+const RTCSignalingState = {
+    stable: 'stable',
+    haveLocalOffer: 'have-local-offer',
+    haveRemoteOffer: 'have-remote-offer',
+    closed: 'closed'
+};
+
+const RTCIceGatheringState = {
+    new: 'new',
+    gathering: 'gathering',
+    complete: 'complete'
+};
+
+/**
+ * RTCPeerConnection shim for ORTC based endpoints (such as Edge).
+ *
+ * The interface is based on the W3C specification of 2015, which matches
+ * the implementation of Chrome nowadays:
+ *
+ *   https://www.w3.org/TR/2015/WD-webrtc-20150210/
+ */
+export default class ortcRTCPeerConnection extends yaeti.EventTarget {
+    /**
+     */
+    constructor(pcConfig) {
+        super();
+
+        logger.debug('constructor() pcConfig:', pcConfig);
+
+        // Closed flag.
+        this._closed = false;
```

Comment:
fields should have their types documented as follows:
`/**`
`* Closed flag.`
`* @type {boolean}`
`**`